src/Data/YamlSymfony.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Kirby\Data;
+
+use Kirby\Toolkit\A;
+use Symfony\Component\Yaml\Yaml as Symfony;
+
+/**
+ * Simple Wrapper around the Symfony YAML class
+ *
+ * @package   Kirby Data
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://opensource.org/licenses/MIT
+ */
+class YamlSymfony
+{
+	/**
+	 * Converts an array to an encoded YAML string
+	 */
+	public static function encode($data): string
+	{
+		return Symfony::dump(
+			$data,
+			2,
```

Comment:
This second argument "customizes the level at which the output switches from the expanded representation to the inline one" according to the docs.

Have you tested what happens if we throw a deeply nested array at it? IMO it must not start to use the inline syntax at all considering our deeply nested blueprint structures.

Reply:
Output with nested structure field:

### Before
````
----

Addresses:

- 
  street: A
  zip: B
  city: C
  addresses:
    - 
      street: "1"
      zip: "2"
      city: "3"
    - 
      street: "4"
      zip: "5"
      city: "6"
    - 
      street: "7"
      zip: "8"
      city: "9"

----
````

### After
````
----

Addresses:

-
  street: A
  zip: B
  city: C
  addresses: [{ street: '1', zip: '2', city: '3' }, { street: '4', zip: '5', city: '6' }]

----
````

Reply:
Thanks for testing! I feel like the nested structure still makes more sense for us, considering version control with Git and manual editing by humans.

Is there a way to disable the inline feature completely? Or do we need to pass a higher value like `9999`?

Reply:
I tested it, I couldn't find a better solution than 9999. 

Also thinking out loud, just a idea: do you think it would be a good idea to set custom parameters for yaml handler from config.php?

````php
'yaml' => [
    'handler' => 'symfony',
    'params' => [
        'inline' => 0,
        'indent' => 4
    ],
]
````

Reply:
I'd name it `options`, apart from that 👍👍.